src/components/decline.ts

```diff
@@ -1,6 +1,7 @@
 import { Client as ErisClient } from 'eris';
 import { ComponentContext } from 'slash-create';
 
+import { undi } from '../util/common';
```

Comment:
`undi` is derived from a blargbot tag, which I was responsible for creating. The principle remains the same to generate a consistent user string format and fill in the gaps for the library interfaces.

https://github.com/TinkerStorm/interaction-prototypes/blob/7ea20d2eec1bd98a02068d7074504483a9c0a268/src/util/common.ts#L40-L47